sdk/sdk.go

```diff
@@ -85,6 +86,16 @@ func (sdk *SDK) DeleteService(serviceID string, deleteData bool) error {
 	if err != nil {
 		return err
 	}
+
+	// check if service hash a running instance.
+	_, err = sdk.Instance.GetByService(s.Hash)
+	if err == nil {
+		return errors.New("stop service's instance before deleting")
```

Comment:
Error should contains the instances' hash. Otherwise it will be very complex for the user to stop the instances.